src/settings.ts

```diff
@@ -5,6 +5,8 @@ export class MealSettings {
     meal_plan_note = 'Meal Plan';
     shopping_list_note = 'Shopping List';
 
+    recipe_format = 'RecipeMD';
```

Comment:
Could this be an enum, and maybe defaulted to not use recipemd

Reply:
Changed the default, will read up on JS enums.

Reply:
Changed to use enums. Assuming I've used them correctly.